src/Model.elm

```diff
@@ -265,5 +276,12 @@ update msg model =
                 Err error ->
                     { model | publicTimeline = [], errors = (errorText error) :: model.errors } ! []
 
+        UserAccount account ->
+            let
+                d =
+                    Debug.log "[UserAccount] Loaded " account
```

Comment:
Hmm, did you left this on purpose? I'm usually relying on the debuuger to track these instead.